lte/gateway/python/mypy.ini

```diff
@@ -0,0 +1,21 @@
+[mypy]
+python_version = 3.7
```

Comment:
Here I was not sure, 3.7. was a wild guess.

Reply:
It's 3.8. :)